### DISCUSSION TITLE
How to Make (G)it 🔶 Happen  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#55862] [New to GitHub]
We’re sharing some fun new tutorials to help kick your Git skills into gear this month, but before we do join me in celebrating some awesome community members. 

🌟**Community Superstars:** A round of applause for these community citizens 👏 
* @lostAlpaca88 & @montymahato for both [offering fixes in the discussion](https://github.com/orgs/community/discussions/54863) to styling on GitHub Pages - thank you! 
* @SoundaryaKoutharapu: for [suggesting a user use GitHub Pages in their reply](https://github.com/orgs/community/discussions/55071) to be able to easily link to their code - great idea 💡! 
* @hrishikesh-kocharekar-dev: Not only did they answer [the question in the community] about removing stale branches 🌳, but they offered solutions to prevent stale branches in the first place, nicely done. 

⬆️ **Level up**: Resources to take your skills to new heights while having fun doing so  ☁️
* **Oh My Git** 🎮:  An open source game that introduces players to Git. It is highly interactive, and aims at building intuition for operations like "merging" or "rebasing" branches. Players are guided through the features of Git step by step. [Click here to play the game on their site](https://ohmygit.org/). 
* **Learn Git Branching** 🧑‍🏫: Offers a variety of levels to learning Git from beginner to more challenging in a visual manner - [use this link to play on their website](https://learngitbranching.js.org/). 
* **GitHub Minesweeper** 💣: Who says you have to learn Git alone? This tutorial will take you through professional workflows with a bot as your teammate. [Start learning now on the profy website](https://profy.dev/project/github-minesweeper). 
Have more questions about Git - use our [Programming Help category](https://github.com/orgs/community/discussions/categories/programming-help) to ask and check out our [Getting Started with Git thread](https://github.com/orgs/community/discussions/51393) for more resources. 

📣**Announcements**: The latest updates in the GitHub Universe 🆕
* **Addressing GitHub’s recent availability issues**: GitHub recently experienced several availability incidents, both long running and shorter duration. We have since mitigated these incidents and all systems are now operating normally. [Read on our blog](https://github.blog/2023-05-16-addressing-githubs-recent-availability-issues/) for more details about what caused these incidents and what we’re doing to mitigate in the future.
* **Maintainer Month** ⚡: Maintainers are the backbone of open source projects, and they deserve our appreciation and support. This Maintainer Month, consider sponsoring a maintainer or contributing to an open source project. Learn more at maintainermonth.github.com #MaintainerMonth and [read how in the ReadMe Project](https://github.com/readme/stories/safia-abdalla) @captainsafia leverages her position as a maintainer to empower underrepresented contributors 💪. 
* **GitHub Summerfest 2023** 📅: Join us virtually for a jam-packed event on June 6th with all the favorites you know and love: Demos from subject matter experts, opportunities for Q&A, swag and fun virtual games! [Click here to learn more and register on the Summerfest site!](https://resources.github.com/github-summerfest-2023/?utm_source=linkedin&utm_medium=social&utm_campaign=2023q4-evt-emea-summerfest-2023LI1)
* **Save the Date for GitHub Universe** 🌠: Our annual developer event is back, better, and bigger than ever at San Francisco’s Yerba Buena Center for the Arts and online on November 8-9. [Check out our Universe	website](https://githubuniverse.com/?utm_source=twitter&utm_medium=social&utm_campaign=savetdvid). 

What’s your favorite or most used Git command? Let us know in the comments 🔶.

**Comment by bethelben** [comment]
is there a couch I could follow for head-to-head guidance?' i will be most honored to be your student.

**Reply by queenofcorgis** [reply]
@bethelben Check out our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion and guidance.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by queenofcorgis** [comment]
Our latest installment is live - **[check out the latest check-in](https://github.com/orgs/community/discussions/56962)**!
